From adf3c9ec039dcc45913c2e0272651ac796bf4884 Mon Sep 17 00:00:00 2001
Message-Id: <adf3c9ec039dcc45913c2e0272651ac796bf4884.1693597912.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1693597912.git.sharlatanus@gmail.com>
References: <cover.1693597912.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sun, 6 Aug 2023 14:36:44 +0100
Subject: [PATCH v2 3/9] gnu: Add go-github-com-bmizerany-perks-quantile.

* gnu/packages/golang.scm (go-github-com-bmizerany-perks-quantile): New variable.
---
 gnu/packages/golang.scm | 24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index b569c9edaa..ab5ed46b9f 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9050,6 +9050,30 @@ (define-public go-github-com-bmatcuk-doublestar
 matching and globbing with support for \"doublestar\" patterns.")
     (license license:expat)))
 
+(define-public go-github-com-bmizerany-perks-quantile
+  (package
+    (name "go-github-com-bmizerany-perks-quantile")
+    (version "0.0.0-20230307044200-03f9df79da1e")
+    (source (origin
+              (method git-fetch)
+              (uri (git-reference
+                    (url "https://github.com/bmizerany/perks")
+                    (commit (go-version->git-ref version))))
+              (file-name (git-file-name name version))
+              (sha256
+               (base32
+                "1f2a99v3618bz2mf61iwhdjm3xi1gam6v4apqgcrz71gj7ba9943"))))
+    (build-system go-build-system)
+    (arguments
+     (list #:unpack-path "github.com/bmizerany/perks"
+           #:import-path "github.com/bmizerany/perks/quantile"))
+    (home-page "https://github.com/bmizerany/perks")
+    (synopsis "Library for computing quantiles")
+    (description
+     "Perks contains the Go package @code{quantile} that computes approximate
+quantiles over an unbounded data stream within low memory and CPU bounds.")
+    (license license:bsd-2)))
+
 (define-public go-github-com-dlclark-regexp2
   (package
     (name "go-github-com-dlclark-regexp2")
-- 
2.40.1

